README.md

```diff
@@ -107,6 +107,7 @@ Image: anonaddy/anonaddy:latest
 * `APP_URL`: The URL of your installation
 * `ANONADDY_RETURN_PATH`: Return-path header for outbound emails
 * `ANONADDY_ADMIN_USERNAME`: If set this value will be used and allow you to receive forwarded emails at the root domain
+* `ANONADDY_NON_ADMIN_SHARED_DOMAINS`: If set to false this will prevent any non-admin users from being able to create shared domain aliases at any domains that have been set for `$ANONADDY_ALL_DOMAINS` (default `true`)
```

Comment:
I think we need to put it in https://github.com/anonaddy/docker/blob/fd6c9fba9578499f7704047d628c59556d9dcbc7/rootfs/etc/cont-init.d/13-config-anonaddy.sh#L22

Otherwise it would have no effect.